uberfire-structure/uberfire-structure-backend/src/main/java/org/guvnor/structure/backend/repositories/RepositoryServiceImpl.java

```diff
@@ -389,43 +407,52 @@ public void removeRepositories(final Space space,
             for (final String alias : aliases) {
                 doRemoveRepository(orgUnit,
                                    alias,
-                                   findRepositoryConfig(alias,
-                                                        space.getName()),
+                                   findRepositoryConfig(space.getName(),
```

Comment:
are you sure about this parameter swap right?

Reply:
Yes, I did that change, I didn't like previous order :p